dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/temp/TempFileAPITest.java

```diff
@@ -0,0 +1,44 @@
+package com.dotcms.rest.api.v1.temp;
+
+import com.dotcms.rest.exception.BadRequestException;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import java.io.IOException;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TempFileAPITest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4616566)